examples/cmd/snapshotting/example_demo.go

```diff
@@ -383,48 +400,57 @@ func loadSnapshotSSH(ctx context.Context, socketPath, memPath, snapPath, ipToRes
 
 func main() {
 	// Check for kvm and root access
-	err := unix.Access("/dev/kvm", unix.W_OK)
-	if err != nil {
+	err := unix.Access("/dev/kvm", unix.W_OK) // Check if the program has write access to /dev/kvm
```

Comment:
Most of the code here is self-explanatory and/or understandable with a quick search. I would prefer to have more readable code, than annotate every line of code with comments. Unless there is some complicate logic

Reply:
Thanks for your kind word.